src/Action/Admin/RefundUnitsAction.php

```diff
@@ -36,22 +38,34 @@ final class RefundUnitsAction
 
     private LoggerInterface $logger;
 
+    private CsrfTokenManagerInterface $csrfTokenManager;
+
     public function __construct(
         MessageBusInterface $commandBus,
         Session $session,
         UrlGeneratorInterface $router,
         RefundUnitsCommandCreatorInterface $commandCreator,
-        LoggerInterface $logger
+        LoggerInterface $logger,
+        CsrfTokenManagerInterface $csrfTokenManager,
```

Comment:
As you were adding a new argument, you could use constructor property promotion here:
```suggestion
        private CsrfTokenManagerInterface $csrfTokenManager,
```